### PR TITLE
Added "-old" to my previous keymap folder; added folder matching my GitHub username

### PR DESCRIPTION
**Pull request by tuesdayjohn**
- Added "-old" to my previous keymap folder (insertsnideremarks) to indicate it's no longer in use
- Added folder matching my current GitHub username (TuesdayJohn)

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

### CODE REVIEW

File: keyboards/kinesis/keymaps/TuesdayJohn/config.h [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#ifndef CONFIG_USER_H
```
Comment (noroadsleft) [inline keyboards/kinesis/keymaps/TuesdayJohn/config.h]: If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`?

It's simpler, and less prone to user error, so if you could make this switch?

```suggestion
#pragma once
```

File: keyboards/kinesis/keymaps/TuesdayJohn/config.h [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+#define CONFIG_USER_H
```
Comment (noroadsleft) [inline keyboards/kinesis/keymaps/TuesdayJohn/config.h]:
```suggestion
```

File: keyboards/kinesis/keymaps/TuesdayJohn/config.h [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+
+#include "../../config.h"
```
Comment (noroadsleft) [inline keyboards/kinesis/keymaps/TuesdayJohn/config.h]: Line 5 can be safely deleted. QMK's build script automatically includes all the `config.h` files in the build tree, so this line is redundant.

```suggestion
```

File: keyboards/kinesis/keymaps/TuesdayJohn/config.h [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+// place overrides here
+#define IGNORE_MOD_TAP_INTERRUPT
+#define TAPPING_TERM 175
+#define TAPPING_TOGGLE 2
+
+
+#endif
```
Comment (noroadsleft) [inline keyboards/kinesis/keymaps/TuesdayJohn/config.h]:
```suggestion
```

File: keyboards/kinesis/keymaps/TuesdayJohn/readme.md [hunk 5]
```diff
@@ -0,0 +1,246 @@
...
+```
+,-----------------------------------------------------. ,----------------------------------------------------.
+| ESC |  F1 |  F2 |  F3 |  F4 |  F5 |  F6 |  F7 |  F8 | | F9 | F10 | F11 | F12 | PScr| SLck| Paus|Numpd|Adjst|
+`-----------------------------------------------------' `----------------------------------------------------'
+```
+
+### Colemak
+- Default layer
+- Keys separated by "/" tap for first, hold for second
+- Uses [Space Cadet Shifts](https://beta.docs.qmk.fm/features/feature_space_cadet_shift)
```
Comment (noroadsleft) [inline keyboards/kinesis/keymaps/TuesdayJohn/readme.md]: The page name is actually `feature_space_cadet`; your current link is Error 404.

```suggestion
- Uses [Space Cadet Shifts](https://beta.docs.qmk.fm/features/feature_space_cadet)
```

File: keyboards/kinesis/keymaps/TuesdayJohn/readme.md [hunk 6]
```diff
@@ -0,0 +1,246 @@
...
+                         ,-------+-------+-------| |-------+-------+-------.
+                         | Space | Enter |App/Alt| |  RGUI | Delete|  Bspc |
+                         |   /   |   /   |-------| |-------|   /   |   /   |
+                         |  Fn   | Number|  Bspc | | Enter |Number2|  Fn2  |
+                         `-----------------------' `-----------------------' 
+```	
+	
+### QWERTY
+- Keys separated by "/" tap for first, hold for second
+- Uses [Space Cadet Shifts](https://beta.docs.qmk.fm/features/feature_space_cadet_shift)
```
Comment (noroadsleft) [inline keyboards/kinesis/keymaps/TuesdayJohn/readme.md]:
```suggestion
- Uses [Space Cadet Shifts](https://beta.docs.qmk.fm/features/feature_space_cadet)
```

**Comment by TuesdayJohn** [comment]
Thank you for checking!  The suggested changes have been made.

File: keyboards/kinesis/keymaps/TuesdayJohn/keymap.c [hunk 7]
```diff
@@ -0,0 +1,509 @@
...
+[PSPA] = ACTION_TAP_DANCE_DOUBLE(KC_PSLS, KC_PAST)   // Numpad slash on a single-tap, numpad asterisk on a double-tap
+};
+
+//Aliases for longer keycodes
+enum kinesis_keycodes {
+  COLEMAK = SAFE_RANGE,
+  QWERTY,
+  COLEMKG,
+  QWERTYG,
+  NUMPAD  = TG(_NUMPAD),
```
Comment (drashna) [inline keyboards/kinesis/keymaps/TuesdayJohn/keymap.c]: You could actually use defines here instead, but this is just fine, too.

File: keyboards/kinesis/keymaps/TuesdayJohn/keymap.c [hunk 8]
```diff
@@ -0,0 +1,509 @@
...
+  return update_tri_layer_state(state, _NUMBERS, _NUMBERS2, _ADJUST);
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+	case COLEMAK:
+      if (record->event.pressed) {
+        default_layer_set(1UL << _COLEMAK);
+//        persistent_default_layer_set(1UL << _COLEMAK);
+        layer_off ( _QWERTY);
```
Comment (drashna) [inline keyboards/kinesis/keymaps/TuesdayJohn/keymap.c]: You could just use `layer_move(0)`, I think.

File: keyboards/kinesis/keymaps/TuesdayJohn/rules.mk [hunk 9]
```diff
@@ -0,0 +1,20 @@
+# Build Options
+#   change to "no" to disable the options, or define them in the Makefile in 
+#   the appropriate keymap folder that will get included automatically
+#
+DEFAULT_FOLDER = kinesis/stapelberg
```
Comment (drashna) [inline keyboards/kinesis/keymaps/TuesdayJohn/rules.mk]: Not needed:

```suggestion
```

**Comment by tuesdayjohn** [comment]
I’ll make the fixes and make another PR.  Fixing the folder name is giving me trouble.

File: keyboards/kinesis/keymaps/TuesdayJohn/keymap.c (hunk 8 above)
Comment (tuesdayjohn) [reply keyboards/kinesis/keymaps/TuesdayJohn/keymap.c]: So just using ‘layer

**Comment by noroadsleft** [comment]
Guessing at this:

If you're on Windows, you need to do this in two stages:

```
git mv TuesdayJohn/ tuesdayjohn-mv/
git mv tuesdayjohn-mv/ tuesdayjohn/
```

Windows isn't case-sensitive but macOS and Linux both are. For Windows `TuesdayJohn` is the same as `tuesdayjohn`, but Linux and macOS would both consider those as two different folders.

**Comment by tuesdayjohn** [comment]
That worked perfectly in changing the folder name.  Thank you!

**Pull request closed without merging**